radix-engine-tests/tests/metering.rs

```diff
@@ -200,11 +200,11 @@ fn test_radiswap() {
         + 11988 /* DropLock */
         + 3255 /* DropNode */
         + 3130196 /* Invoke */
-        + 2279411 /* LockSubstate */
+        + 2319706 /* LockSubstate */
         + 18200 /* ReadSubstate */
         + 125000 /* RunNative */
         + 20000 /* RunSystem */
-        + 602635 /* RunWasm */
+        + 613895 /* RunWasm */
```

Comment:
It's impressive how much this can differ depending on where the WASM was last compiled